src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -0,0 +1,590 @@
+Metadata should be included alongside the data in the `.json` and `.tsv` files.
+The current list of allowed data file formats:
+
+<!-- We should define icephys and ecephys -->
+
+| **Format**                                                                          | **Extension(s)** | **Description**                                                                                                                                                                                                           |
+| ----------------------------------------------------------------------------------- | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Neuroscience Information Exchange Format](https://nixio.readthedocs.io/en/latest/) | `.nix`           | A generic and open  framework with an hdf5 backend and a defined interface to many microephys formats via the [Neo library](https://neo.readthedocs.io/en/latest/). The `.nix` file has to contain a valid Neo structure. |
+| [Neurodata Without Borders](https://www.nwb.org)                                    | `.nwb`           | An open data standard for neurophysiology, including data from intracellular and extracellular electrophysiology experiments.                                                                                             |
+
+Both of these formats can also store essential metadata of the datasets.
+Some of this metadata needs to be duplicated in BIDS `.tsv` and `.json` sidecar files.
+Even though the duplication requires additional effort to ensure the consistency of the data, it provides several advantages:
+
+-   It makes the dataset easier for humans to scan, as essential information is easily accessible without loading the data files.
+-   The dataset adheres to the BIDS standard and can benefit from tools built on top of this standard, such as [bids-validator](https://github.com/bids-standard/bids-validator).
+-   It simplifies the separation of data and basic metadata, enabling, for example, the publication of a dataset in a lightweight fashion with access to the data files on request (as implemented by [DataLad](https://www.datalad.org)).
+
+<!--
+This block generates a filename templates.
+The inputs for this macro can be found in the directory
+  src/schema/rules/files/raw
+and a guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+
+<!-- Link Definitions -->
+
+### icephys
+{{ MACROS___make_filename_template(
+   "raw",
+   datatypes=["icephys"],
+   suffixes=["icephys", "events", "channels", "electrodes","scans","probes","coordsystem"]
+)
+}}
+
+### ecephys
+{{ MACROS___make_filename_template(
+   "raw",
+   datatypes=["ecephys"],
+   suffixes=["ecephys", "events", "channels", "electrodes","scans","probes","coordsystem"]
```

Comment:
in both types of files we might have `_photo.jpg` -- may be should be added here too?